cypress/integration/chi-custom-elements/date-picker.spec.js

```diff
@@ -38,6 +39,49 @@ describe('Date picker', function() {
       .then(() => {
         expect(spy).to.be.called;
         expect(spy.getCall(0).args[0].detail).to.equal(clickDate);
+        expect(spy.getCall(0).args[0].target.nodeName).to.equal('CHI-DATE');
+      });
+  });
+
+  it(`Changing the input's date value emits the ${chiDateChange} event`, function() {
+    const date = '12/02/2021';
+    const spy = cy.spy();
+
+    cy.get('[data-cy="test-input-combined-picker"]').then(el => {
+      el.on(chiDateChange, spy);
+    });
+
+    cy.get('[data-cy="test-input-combined-picker"]')
+      .find('input')
```

Comment:
Can you try this?
```suggestion
    cy.get('@aliasName input')
```

Reply:
Doesn't work sadly :(

Reply:
:( is input the first children? maybe @aliasName > input does the trick. If not..it's ok..leave it as it was. Thanks for trying!

Reply:
Nope :( it says "Could not find registered alias for @picker > input", or before, "@picker input", it only lets me do cy.get("@picker")

Reply:
Interesting..I know it works using something like this: cy.get('[data-cy=cy-tag] div') but it seems it doesn't using aliases. It's ok. Thanks again!